qemu/tests/cfg/trim_support_test.cfg

```diff
@@ -15,7 +15,6 @@
     image_readonly_stg1 = no
     drive_index_stg1 = 3
     image_boot_stg1 = no
-    blk_extra_params_stg1 = "discard=on"
     drv_extra_params_stg1 = "discard=unmap"
```

Comment:
I remember some situation for virtio-blk, should add "discard=on" for it. So if virtio-scsi not use it, you can set this to virtio-blk only.

Reply:
updated